_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -0,0 +1,188 @@
+      to messages-over-payments, onion messages have several advantages:
+
+        1. They have a [draft specification][onion messages draft spec]
+           which, if adopted, will make it easier for multiple
+           implementations to support them.
+
+        2. They don't need the security of an onchain-enforceable payment
+           channel so onion messages can be routed even between peers
+           that don't share an established payment channel.  <!-- BOLT7
+           draft PR: "SHOULD accept onion messages from peers without an
+           established channel." -->
+
+        3. They don't require the bidirectional transmission of
+           information like HTLCs or error messages, so once a node has
+           forwarded a message, it doesn't need to keep any information
+           related to that message.  This minimizes the memory
+           requirements for nodes.  If the sending node wants to receive
+           a reply, the draft specification allows it to include a
+           blinded `reply_path` field that the receiving node can use to
+           send a reply in a new message.
+
+    - *Blinded paths* (called "lightweight rendez-vous routing" in
+      [Newsletter #85][news85 lw rv]) make it [possible][blinded path
+      gist] to route a payment or a message without the originator
+      learning the destination's network identity or the full path used.
+      This is accomplished through several steps:
+
+        1. The destination node chooses a path from an intermediate node
+           to itself and then onion-encrypts that path information so
+           that each hop in the path will only be able to decrypt the
+           identifier for the next node that should receive the message.
+           The destination node gives this encrypted ("blinded") path
+           information to the sending node (e.g. via a field in a
+           [BOLT11][] invoice or using the previously-mentioned onion
+           message `reply_path` field).
+
+        2. The sending node relays its message using normal onion
+           routing to the intermediate node.
+
+        3. The intermediate node decrypts the next hop to use from the
+           blinded path and sends the message to it.  The next node
+           decrypts its own next hop field and further relays the
+           message; this process continues until the message reaches the
+           destination node.
+
+      Just as with normal onion routing, no routing node should learn
+      more about the blinded path than which node sent them the message
+      and which node should receive the message after them.  Unlike with
+      normal routing, neither the origination nor destination node needs
+      to learn the identity of the other node or what exact path it
+      used.  This improves not just the privacy of those endpoints but
+      also the privacy of any unannounced nodes along the blinded paths.
```

Comment:
It might be worth mentioning that blinded paths would provide more flexibility (reusability and retrying with updated fees/amounts/locktimes), at the cost of somewhat less privacy, compared to rendez-vous.

Reply:
I don't see a good way to add this.  In an earlier draft, I explained the difference between RV routing and blinded paths, but I deleted it because it was too detail in this already over-detailed PR description.  Without our clarifying the difference between RV and blinding (which is kinda subtle), I think comparing the two would just be confusing.  Hopefully we'll find another chance in the near future to describe the difference and the tradeoffs.